pkg/datastore/solr.go

```diff
@@ -11,6 +11,8 @@ import (
 	"go.opencensus.io/plugin/ochttp"
 )
 
+const schema, update = "/schema", "/update"
```

Comment:
Shouldn't name be much more descriptive?

Reply:
can be prefixed with `solr` ?

Reply:
renamed that to `updateSchema`